helm/values.yaml

```diff
@@ -66,6 +66,7 @@ env:
       fixGeometry:
         enabled: true
         bufferSizeDeg: 0.000006
+        bufferStyleParameters: ''
```

Comment:
Please add comment with available values or link to the st_buffer docs

Reply:
added